app/views/comments/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<p style="color: green"><%= notice %></p>
+
```

Comment:
small nitpick: Feel free to move your h1 tag up to line 1; otherwise, everything looks great!